host_v2/src/containers/GameSessionContainer.tsx

```diff
@@ -17,10 +18,15 @@ export default function GameSessionContainer({apiClients, backendGameSession, ba
   const [localGameSession, dispatch] = useReducer(GameSessionReducer, backendGameSession);
   const [localHostTeamAnswers, setLocalHostTeamAnswers] = React.useState<IHostTeamAnswers>(backendHostTeamAnswers);
   useEffect(() => {
+    console.log('backendHostTeamAnswers', backendHostTeamAnswers);
     setLocalHostTeamAnswers(backendHostTeamAnswers);
   }, [backendHostTeamAnswers]);
   const handleDeleteTeam = () => {};
 
```

Comment:
useEffect synchs updates from subscription updates coming from the custom hook, into the localGameSession held in context.